pkg/causality/internal/node.go

```diff
@@ -212,96 +180,41 @@ func (n *Node) assignTo(workerID int64) bool {
 	defer n.mu.Unlock()
 
 	if n.assignedTo != unassigned {
-		// Already resolved by some other guys.
+		// Already handled by some other guys.
```

Comment:
// Already assign to a worker by another goroutrine.

Reply:
`handled` is a generic word here, not equal to `executed`.